### DISCUSSION TITLE
openimageio x64 download hash error

### DISCUSSION THREAD

**Discussion by aforanup** [microsoft/vcpkg#35921] [Q&A]
while installing xstudio downloading package openimageio from https://github.com/OpenImageIO/oiio/archive/v2.4.11.1.tar.gz
gives error

`-- Downloading https://github.com/OpenImageIO/oiio/archive/v2.4.11.1.tar.gz -> OpenImageIO-oiio-v2.4.11.1.tar.gz...
[DEBUG] To include the environment variables in debug output, pass --debug-env
[DEBUG] Disabling metrics because vcpkg.disable-metrics exists
[DEBUG] Trying to load bundleconfig from d:\xstudio-windows\xstudio-windows\build\vcpkg\vcpkg-bundle.json
[DEBUG] Failed to open: d:\xstudio-windows\xstudio-windows\build\vcpkg\vcpkg-bundle.json
[DEBUG] Bundle config: readonly=false, usegitregistry=false, embeddedsha=nullopt, deployment=Git, vsversion=nullopt
[DEBUG] Feature flag 'binarycaching' unset
[DEBUG] Feature flag 'compilertracking' unset
[DEBUG] Feature flag 'registries' unset
[DEBUG] Feature flag 'versions' unset
[DEBUG] Feature flag 'dependencygraph' unset
Downloading https://github.com/OpenImageIO/oiio/archive/v2.4.11.1.tar.gz
[DEBUG] Trying to hash d:\xstudio-windows\xstudio-windows\build\vcpkg\downloads\OpenImageIO-oiio-v2.4.11.1.tar.gz.16372.part
[DEBUG] d:\xstudio-windows\xstudio-windows\build\vcpkg\downloads\OpenImageIO-oiio-v2.4.11.1.tar.gz.16372.part has hash 5e7710676eb2bd97fa8d30ac3f9fab0bf95e4c71389f41694290b423712b64944b15135d3929cfb72744c6d5ff0ad3a85d3c1d280a439637639ef42ff10203b7
error: Failed to download from mirror set
error: File does not have the expected hash:
url: https://github.com/OpenImageIO/oiio/archive/v2.4.11.1.tar.gz
File: d:\xstudio-windows\xstudio-windows\build\vcpkg\downloads\OpenImageIO-oiio-v2.4.11.1.tar.gz.16372.part
Expected hash: e8b232bb3c1bb66cc6c4f023dcf6e29633a1aee64c49a2860c2157c0885960c40114ee3988d4f132e6e55670b8b1e01b5b4cd4462651ae047a89d22de527581c
Actual hash: 5e7710676eb2bd97fa8d30ac3f9fab0bf95e4c71389f41694290b423712b64944b15135d3929cfb72744c6d5ff0ad3a85d3c1d280a439637639ef42ff10203b7
[DEBUG] D:\a\_work\1\s\src\vcpkg\base\downloads.cpp(1048): 
[DEBUG] Time in subprocesses: 0us
[DEBUG] Time in parsing JSON: 6us
[DEBUG] Time in JSON reader: 0us
[DEBUG] Time in filesystem: 293us
[DEBUG] Time in loading ports: 0us
[DEBUG] Exiting after 6.5 s (6469982us)

CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:32 (message):
      
      Failed to download file with error: 1
      If you are using a proxy, please check your proxy setting. Possible causes are:
      
      1. You are actually using an HTTP proxy, but setting HTTPS_PROXY variable
         to `https://address:port`. This is not correct, because `https://` prefix
         claims the proxy is an HTTPS proxy, while your proxy (v2ray, shadowsocksr
         , etc..) is an HTTP proxy. Try setting `http://address:port` to both
         HTTP_PROXY and HTTPS_PROXY instead.
      
      2. If you are using Windows, vcpkg will automatically use your Windows IE Proxy Settings
         set by your proxy software. See https://github.com/microsoft/vcpkg-tool/pull/77
         The value set by your proxy might be wrong, or have same `https://` prefix issue.
      
      3. Your proxy's remote server is out of service.
      
      If you've tried directly download the link, and believe this is not a temporary
      download server failure, please submit an issue at https://github.com/Microsoft/vcpkg/issues
      to report this upstream download server failure.
      

Call Stack (most recent call first):
  scripts/cmake/vcpkg_download_distfile.cmake:270 (z_vcpkg_download_distfile_show_proxy_and_fail)
  scripts/cmake/vcpkg_from_github.cmake:106 (vcpkg_download_distfile)
  buildtrees/versioning_/versions/openimageio/7c06030b088b87143ff7a85c23af606bbd967496/portfile.cmake:1 (vcpkg_from_github)
  scripts/ports.cmake:170 (include)`

**Comment by StarGate-One** [comment]
Please open under issues instead as a discussion/question - See [How to report a package build failure [explanation video] #30604](https://github.com/microsoft/vcpkg/issues/30604).
